app/manifest.json

```diff
@@ -42,15 +42,7 @@
       "run_at": "document_idle"
     }
   ],
-  "applications": {
```

Comment:
This code is used for firefox compatibility. Don't remove it.

Reply:
Ok mais cela crée des erreurs sous Chrome

Reply:
J'ai corrigé tous tes commentaires, tiens moi au courant ;) Merci!